src/components/generators/view-components/string.vue

```diff
@@ -54,4 +97,38 @@ export default {
 .fa-external-link:hover {
   cursor: pointer;
 }
+
+.toast-container {
+  display: block;
+  max-width: 350px;
+  backface-visibility: hidden;
+  -webkit-backface-visibility: hidden;
+  background-clip: padding-box;
+  z-index: 1;
+  border-radius: 0.25rem;
+}
+
+.toast-container-top-center {
+  position: fixed;
+  min-height: 200px;
```

Comment:
rems :)

Reply:
It wasn't actually needed the min-height so I deleted it